src/platform/site-wide/va-footer/helpers.jsx

```diff
@@ -31,6 +31,7 @@ const renderInnerTag = (link, captureEvent) => (
     ) : null}
     {link.href ? (
       <a
+        aria-label={link.ariaLabel}
```

Comment:
At first I was wondering what would happen if an `ariaLabel` weren't provided, but then I see that it just doesn't add that attribute to the DOM node, so this looks great! 🙂